### PR TITLE
docs: update notes to update conan packages

### PR DESCRIPTION
**Pull request by coryan**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/12606)

### CODE REVIEW

File: release/README.md [hunk 1]
```diff
@@ -200,6 +200,16 @@ next 24 hours.
 (Optional) Create a PR to update the `google-cloud-cpp` package in
 [Conan](https://conan.io). These PRs are more involved.
 
+This package manager uses CMake, but ignores the library and target definitions
+exported by our CMake files. It reimplement the library definitions and
```
Comment (devbww) [inline release/README.md:204]: s/reimplement/reimplements/

File: release/README.md [hunk 2]
```diff
@@ -200,6 +200,16 @@ next 24 hours.
...
+dependencies in Python code. The recipes for each package does this for every
```
Comment (devbww) [inline release/README.md:205]: s/does/do/

File: release/README.md [hunk 3]
```diff
@@ -200,6 +200,16 @@ next 24 hours.
...
+package (gRPC, Protobuf, Boost, etc.) not just for our package recipe. For most
+packages, where there are only a handful of libraries, this is trivial, but our
+package has about 240 libraries (including the `*_protos` libraries).
+
+We use a helper script to automatically generate the Python data structures that
```
Comment (devbww) [inline release/README.md:210]: Join these two paragraphs with "So, we ..."?

File: release/README.md [hunk 4]
```diff
@@ -200,6 +200,16 @@ next 24 hours.
 (Optional) Create a PR to update the `google-cloud-cpp` package in
 [Conan](https://conan.io). These PRs are more involved.
 
+This package manager uses CMake, but ignores the library and target definitions
+exported by our CMake files. It reimplement the library definitions and
+dependencies in Python code. The recipes for each package does this for every
+package (gRPC, Protobuf, Boost, etc.) not just for our package recipe. For most
```
Comment (devbww) [inline release/README.md:206]: Too many "package"s in that sentence?

**Review by devbww — APPROVED** [review]

File: release/README.md (hunk 1 above)
Comment (coryan) [reply release/README.md]: Fixed.

File: release/README.md (hunk 2 above)
Comment (coryan) [reply release/README.md]: Fixed.

File: release/README.md (hunk 3 above)
Comment (coryan) [reply release/README.md]: Fixed.

File: release/README.md (hunk 4 above)
Comment (coryan) [reply release/README.md]: Maybe better now?

File: release/README.md [hunk 5]
```diff
@@ -200,6 +200,15 @@ next 24 hours.
 (Optional) Create a PR to update the `google-cloud-cpp` package in
 [Conan](https://conan.io). These PRs are more involved.
 
+This package manager uses CMake, but ignores the library and target definitions
+exported by our CMake files. It reimplements the library definitions and
+dependencies in Python code. The recipes do this for every package (gRPC,
+Protobuf, Boost, etc.), not just for `google-cloud-cpp`. For most packages,
+where there are only a handful of libraries, this is trivial. `google-cloud-cpp`
+has about 240 libraries (including the `*_protos` libraries). So, we use a
```
Comment (devbww) [inline release/README.md:208]: Looks good.

My only final suggestion would be, _"But `google-cloud-cpp` has ..., so we use ..."_.

Comment (coryan) [reply release/README.md]: Done.

**Pull request merged**